adapters/Vungle/VungleAdapter/VungleRouterConsent.m

```diff
@@ -10,23 +10,20 @@
 #import <VungleSDK/VungleSDK.h>
 
 static VungleConsentStatus currentConsentStatus = 0;
-static NSString *currentConsentMessageVersion = @"";
 
 @implementation VungleRouterConsent
-+ (void)updateConsentStatus:(VungleConsentStatus)status consentMessageVersion:(NSString *)version {
-  currentConsentStatus = status;
-  currentConsentMessageVersion = version;
++ (BOOL)updateConsentStatus:(VungleConsentStatus)consentStatus {
   VungleSDK *sdk = [VungleSDK sharedSDK];
-  if (sdk.initialized && status > 0) {
-    [sdk updateConsentStatus:status consentMessageVersion:version];
+  if (consentStatus > 0) {
+    currentConsentStatus = consentStatus;
+    [sdk updateConsentStatus:consentStatus consentMessageVersion:@""];
```

Comment:
Why are we sending an empty string as the consentMessageVersion? Is it not needed for the publisher to set this value?

Reply:
@maninomula The same as above. Since Vungle uses `consentMessageVersion` for only native integration case, we decided to hide in mediation adapter level.